src/utils/miscellaneous.ts

```diff
@@ -13,7 +13,28 @@ export function assertNonNull<T>(val: T, message: string): asserts val is NonNul
 function noop(..._args: any[]): void {}
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const log = process.env.NODE_ENV === 'test' ? noop : (...args: any[]): void => console.log(...args)
+export const logInfo = process.env.NODE_ENV === 'test' ? noop : (...args: any[]): void => console.log(...args)
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+let debugEnabled = process.env.NODE_ENV === 'development'
+
+declare global {
+  interface Window {
+    toggleDebug: () => void
```

Comment:
this should be
```suggestion
    toggleDebug: () => boolean
```
no?

Reply:
I had made that change in a different PR. So it's now in this one (I've merged develop here)